projects/fsastorefrontstyles/scss/overrides/payment-details.scss

```diff
@@ -3,10 +3,18 @@
 
 .cx-msg {
   margin-bottom: 30px;
+  font-size: var(--cx-h3-font-size);
+  font-weight: var(--cx-font-weight-semi);
 }
 .cx-header h3 {
   @include headline-style;
   padding: 1rem 0;
   margin: 1rem auto 3rem;
   text-align: center;
 }
+
+%cx-payment-methods {
+  .cx-header {
+    padding: 0;
+  }
+}
```

Comment:
This file should be renamed to `payment-methods` to reflect what is being overridden. Also `%cx-payment-methods` should wrap all the rules. `.cx-header` is already declared in this file, so just override it there